content/docs/reference/clouds/gcp/_index.md

```diff
@@ -32,12 +32,32 @@ In addition to the tutorial, several interesting examples are available complete
 
 ## Example
 
+{{< langchoose >}}
+
 ```javascript
 const gcp = require("@pulumi/gcp")
 
 const bucket = new gcp.storage.Bucket("my-bucket");
 ```
 
+```typescript
+const gcp = require("@pulumi/gcp")
+
+const bucket = new gcp.storage.Bucket("my-bucket");
+```
+
+```python
+from pulumi_aws import s3
```

Comment:
Should this be `pulumi_gcp` to match the other languages?

Reply:
yup this should be `from pulumi_gcp import storage`